index.html

```diff
@@ -3,17 +3,91 @@
-</body>
+      <div class="nav-container">
+        <header class="nav-content">
+          <nav>
+            <a href="services.html" class="nav-button">Services</a>
+            <a href="#" class="nav-button">Product</a>
+            <a href="#" class="nav-button">Vision</a>
+            <a href="#" class="nav-button">Features</a>
+            <a href="#" class="nav-button">About</a>
+            <a href="#" class="nav-button">Contact</a>
+          </nav>
+          <a class="logo" href="index.html"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
+        </header>
+      </div>
+
+      <section class="call-to-action">
+        <div class="cta-text">
+          <h1 class="innovation">
+            Innovation<br>
+            On<br>
+            Demand
+          </h1>
+          <div class="get-started">
+            <a href="">Get Started</a> 
+          </div>
+        </div>
+        <img class="cta-img"src="img/header-img.png" alt="Image of a code snippet.">
+      </section>
+
+      <section class="top-content">
+        <div class="features">
+          <h2>Features</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+        <div class="about">
+          <h2>About</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+      </section>
+
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+      <section class="bottom-content">
+        <div class="services">
+          <h2>Services</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+        <div class="product">
+          <h2>Product</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+        <div class="vision">
+          <h2>Vision</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+      </section>
+
+      <section class="contact">
+        <address>
+          <h2>Contact</h2>
+          <p>123 Way 456 Street <br>
+          Somewhere, USA</p>
+          <p><a href="tel:18888888888">1 (888) 888-8888</a></p>
```

Comment:
```suggestion
          <a href="tel:18888888888"><a href="tel:18888888888">1 (888) 888-8888</a></a>
```
Putting this in an anchor tag will tell the browser that this is a phone number. Makes for a better UI experience. You can also do something similar with the email address on the line below to tell the browser that it is an email address.

Reply:
Use a double anchor? Did you mean this:

```suggestion
          <a href="tel:18888888888">1 (888) 888-8888</a>
```